vars/buildPlugin.groovy

```diff
@@ -161,7 +161,17 @@ def call(Map addonParams = [:])
 							dir("tools/depends/target/binary-addons")
 							{
 								if (isUnix())
-									sh "make -j $BUILDTHREADS ADDONS='${addon}' ADDONS_DEFINITION_DIR=`pwd`/addons ADDON_SRC_PREFIX=`pwd` EXTRA_CMAKE_ARGS=\"-DPACKAGE_ZIP=ON -DPACKAGE_DIR=`pwd`/../../../../cmake/addons/build/zips\" PACKAGE=1"
+								{
+									def extraCmakeArgs = [
+										'PACKAGE_ZIP=ON',
+										"PACKAGE_DIR=`pwd`/../../../../cmake/addons/build/zips",
+									]
+									if (platform == 'osx-x86_64' && version == 'Leia')
```

Comment:
Why Leia only? Will newer Kodi versions not get signed anymore?
Does setting CODE_SIGN_IDENTITY for the other platforms break the build?

Reply:
like I wrote in the related PR, Leia releases are already signed, while nightly builds aren't yet. The version condition will be removed in the next PR when Kodi builds become signed.

CODE_SIGN_IDENTITY doesn't break other platforms, but why add an unused variable? Should I remove the platform condition still?

Reply:
Why make the code more complex then required?
I would actually prefer if there is no change required in the `buildPlugin` function and handle it in Kodi code only.

Reply:
> I would actually prefer if there is no change required in the buildPlugin function and handle it in Kodi code only.

could you give a hint how to pass the signing identity as a parameter in such case?  Basically signing should be done only by Jenkins, development builds don't care about code signing. But still it should be possible to pass the signing identity manually (e.g. for testing on local machine). That's why I thought it'd be natural to modify the actual build command.

Reply:
e.g. only pass `-DSIGN:BOOL=ON` and handle everything else in `cmake/addons` or ` cmake/scripts/common/AddonHelpers.cmake`